docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -449,7 +453,28 @@ Simulant will be randomly assigned to a parent/guardian based on the below rules
 #. The foster care system is complex. We have the foster kid assigned within the house they are currently living. If we model the foster care system in more detail, we might improve this at some point. 
 #. We have "parents" fall between 20-45 older than the child. This is an oversimplification. Some parents (especially men) fall outside of this range. Also some age gaps are more common than others. 
 #. The only people who are seen as "in college" are in GQ in college. Plenty of people attend college from home, but we do not track education so are not accounting for this. 
-#. We assign GQ college folks to "parents" instead of "parents/guardians". Some are likely supported by a grandparent or other person outside of our qualifications, but this is not included. 
+#. We assign GQ college folks to "guardians" within an age limit. Some are likely supported by a grandparent or other person outside of our qualifications, but this is not included. 
+
+Guardian(s) on Time Steps
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For those who had a guardian initialized at the start of the 
+simulation, that assignment will not change. If the guardian 
+moves, they will remain as the guardian. If they die, the 
+simulant will not have a living guardian. 
+
+A person cannot "age out" of their guardian when they become 
+an adult or have their own children. Therefore it is possible 
+to both HAVE a guardian and BE a guardian. 
+
+However, for simulants born in the simulation, they receive a 
+tracked mother (term for the person that births them, regardless 
+of gender). 
+
+The tracked mother will be assigned as a guardian. In addition, if 
```

Comment:
I think it is good to have this here, but we might want to have it in the fertility section as well? Or reference it from that section? From an engineering perspective, this will be implemented in that component, I believe.

Reply:
Hmm good point, I'll add that in!